config/default.js

```diff
@@ -27,6 +27,11 @@ module.exports = {
     from: 'dev@example.com',
     path: `${__dirname}/mailer`,
   },
+  'auth-orcid': {
+    clientID: '',
+    clientSecret: '',
+    sandbox: true,
```

Comment:
I'd prefer to have the possibility to configure any URL, as in
https://github.com/elifesciences/profiles-formula/blob/master/salt/pillar/profiles.sls#L11
where we can configure the real ORCID, the sandbox ORCID, or a [dummy](https://github.com/elifesciences/orcid-dummy). `https://orcid.org/` is probably fine-grained enough

Reply:
Good point. But bear in mind the whole ORCID login functionality is being removed imminently, see #636

Reply:
@giorgiosironi ... Is this okay then as its just for the short-term?